.travis.yml

```diff
@@ -656,10 +656,12 @@ install:
       export TRAVIS_PYTHON_VERSION=3.7.2;
     fi
   - source $SPACK_ROOT/share/spack/setup-env.sh
+  - export SPACK_ARCH=$(spack spec --json cmake $CXXSPEC | jq -r '.spec[0].cmake.arch| (.platform + "-" + .platform_os + "-x86_64")')
   # fresh (cache-cleaned) travis runs seem to not properly init environment-modules
   - echo $MODULEPATH
   - if [[ ! $MODULEPATH = *"spack"* ]]; then
-      export MODULEPATH=$SPACK_ROOT/share/spack/modules/$(spack arch):$MODULEPATH;
+      export MODULEPATH=$SPACK_ROOT/share/spack/modules/${SPACK_ARCH}:$MODULEPATH;
```

Comment:
@alalazo this makes me also nervous, after sourcing the spack environment modules for various micro-archs are not activated on fresh spack installs.